backend/ipmi.pm

```diff
@@ -142,6 +149,57 @@ sub get_mc_status {
     $self->ipmitool("mc selftest");
 }
 
+sub do_mc_reset {
+    my ($self) = @_;
+
+    # deactivate sol console before doing mc reset because it breaks sol connection
+    if (defined $testapi::distri->{consoles}->{'sol'}) {
+        bmwqemu::diag("Before doing mc reset, sol console exists, so cleanup it");
+        $testapi::distri->{consoles}->{'sol'}->reset();
```

Comment:
the travis job fails because you use useless quotes here. Just {sol} will do

Reply:
got it, will change.